events/masterPanels.js

```diff
@@ -1,66 +1,19 @@
 const db = require("quick.db")
```

Comment:
no xD
is it possible that rebasing went a bit wrong?

Reply:
I'll look into this

Reply:
um what happened with this? I don't see anything wrong